src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -311,61 +321,59 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     [asset, accountIdsByChainId],
   )
 
-  const assetBalanceCryptoBaseUnit = useAppSelector(state =>
+  const poolAssetBalanceCryptoBaseUnit = useAppSelector(state =>
     selectPortfolioCryptoBalanceBaseUnitByFilter(state, assetBalanceFilter),
   )
+  const poolAssetFeeAsset = useAppSelector(state => selectFeeAssetById(state, asset?.assetId ?? ''))
```

Comment:
Note `poolAsset` terminology - we will still need to do RUNE gas estimations, with the exact same logic on the RUNE side but that's not related to this PR